dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -47,6 +47,8 @@ repertiore.HIGH=Cipher suites 3DES and 128 bit and higher
 repertiore.MEDIUM=Cipher suites DES and 40 bit
 repertiore.LOW=Cipher suites without encryption
 repertiore.CUSTOM=Custom list of cipher suites
+repertoire.verifyHostname=Perform host name verification for outbound SSL connections using a specific SSL configuration
```

Comment:
Change
"Perform host name verification for outbound SSL connections using a specific SSL configuration"
to
"The verifyHostname attribute performs host name verification for outbound SSL connections using a specific SSL configuration."

Reply:
@Charlotte-Holt I've update the metetype.properties file with your suggested changes.  Thank you for you comments.

Reply:
@acdemyers Label added. Thank you!